object_store/src/lib.rs

```diff
@@ -153,19 +153,22 @@ impl GoogleCloudStorage {
             }
         );
 
-        let location = location.to_string();
+        let location_string = location.to_string();
```

Comment:
From a style point of view, I don't think we'd normally have a type as part of the variable name. Thoughts @e-dard or @alamb?

Reply:
Agree. Should be `location`.

Reply:
Ah right I forgot to call this out. I made it a new variable because `location` was previously being used in tokio spawn blocking closure and the errors required the `location` variable as well. Would like guidance on how to best solve this issue.

Reply:
Right -- in this case it seems like we need to make two owned things (one for use after `await` and another for the closure). 

My simplest suggestion is to simply rename it to something that makes it clear a copy is needed to pass to the closure:

```suggestion
        let location_copy = location.to_string();
```

An alternate would be to make the copy in an inner closure:
```rust
let location = location.string();
{ 
  let location = location.clone();
  let _ = tokio::task::spawn(...)
  ...
}
```

Reply:
Updated to `location_copy`